### DISCUSSION TITLE
How do we retain logs for more than 90 days?

### DISCUSSION THREAD

**Discussion by saurabhsrivastava2009** [reportportal/reportportal#2030] [Q&A] [answered]
Hi Team,

I was going through the retention policy and saw that we cannot increase the Logs and attachments retention for more than 90 days. Is there a way to increase the number of days?

Thanks.

**Comment by DzmitryHumianiuk** [comment, marked as answer]
@saurabhsrivastava2009 Yes, it's possible to set it up to `FOREVER`. 
Please experiment with all three configurations in the KEEP data section. 

The time period for the launch is the primary consideration, and you can't set the logs and attachments to be kept longer than the launch itself. 

<img width="568" alt="image" src="https://github.com/reportportal/reportportal/assets/11332788/0d06632c-244d-46db-ab25-641d6da723a7">
